ui/shared/QrPopup.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtGraphicalEffects 1.13
+import "../imports"
+
+Popup {
```

Comment:
Would it be possible to use `ModalPopup` instead since it shares the same Overlay, radius, etc?
Or does the header or footer get in the way?

Reply:
when you leave the header of a ModalPopup empty it leaves a seperator under the header, that's not configured to invisible

Reply:
Ah ok, would it be possible to change ModalPopup to be smarter and hide the header completely when it is null?

Reply:
ok ill try that out!

Reply:
Done!